tests/test_timing_properties.py

```diff
@@ -0,0 +1,143 @@
+            del task.timing.samp_quant_samp_mode
+            assert task.timing.samp_quant_samp_mode == AcquisitionType.CONTINUOUS
+
+    def test_float64_property(self, any_x_series_device):
+        """Test for validating float64 property."""
+        with nidaqmx.Task() as task:
+            task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+
+            task.timing.cfg_samp_clk_timing(1000)
+
+            # Test property initial value.
+            assert task.timing.samp_clk_rate == 1000
+
+            # Test property setter and getter.
+            task.timing.samp_clk_rate = 2000
+            assert task.timing.samp_clk_rate == 2000
+
+            # Test property deleter.
+            del task.timing.samp_clk_rate
+            assert task.timing.samp_clk_rate == 1000
+
+    def test_uint32_property(self, any_x_series_device):
+        """Test for validating unsigned int32 property."""
+        with nidaqmx.Task() as task:
+            task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+
+            task.timing.cfg_samp_clk_timing(1000)
+
+            # Test property initial value.
+            default_value = task.timing.samp_clk_timebase_div
+            assert task.timing.samp_clk_timebase_div == default_value
+
+            # Test property setter and getter.
+            value_to_test = random.randint(500, 10000)
+            task.timing.samp_clk_timebase_div = value_to_test
+            assert task.timing.samp_clk_timebase_div == value_to_test
+
+            # Test property deleter.
+            del task.timing.samp_clk_timebase_div
+            assert task.timing.samp_clk_timebase_div == default_value
+
+    def test_uint64_property(self, any_x_series_device):
+        """Test for validating unsigned int64 property."""
+        with nidaqmx.Task() as task:
+            task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+
+            default_value = task.timing.samp_quant_samp_per_chan
+
+            task.timing.cfg_samp_clk_timing(1000, samps_per_chan=100)
+
+            # Test property initial value.
+            assert task.timing.samp_quant_samp_per_chan == 100
+
+            # Test property setter and getter.
+            value_to_test = random.randint(500, 10000)
+            task.timing.samp_quant_samp_per_chan = value_to_test
+            assert task.timing.samp_quant_samp_per_chan == value_to_test
+
+            # Test property deleter.
+            del task.timing.samp_quant_samp_per_chan
+            assert task.timing.samp_quant_samp_per_chan == default_value
+
+            # Test property with invalid value(Minimum value is 2).
+            # Reading this property throws DaqmxError.
+            task.timing.cfg_samp_clk_timing(1000, samps_per_chan=1)
+
+            with pytest.raises(DaqError) as e:
+                _ = task.timing.samp_quant_samp_per_chan
+            assert e.value.error_code == -200077
```

Comment:
-200077 is `DAQmxErrors.INVALID_ATTRIBUTE_VALUE` from `nidaqmx.error_codes`

Reply:
Updated `constants` in place of values.